llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1103,6 +1104,16 @@ void AsmPrinter::emitBBAddrMapSection(const MachineFunction &MF) {
     // always be computed from their offsets.
     emitLabelDifferenceAsULEB128(MBB.getEndSymbol(), MBBSymbol);
     OutStreamer->emitULEB128IntValue(getBBAddrMapMetadata(MBB));
+    // Emit the index of the corresponding LLVMIR basic block.
+    size_t i = 0;
```

Comment:
Is `size_t` the right type here? It *might* be, but it's not obvious to me.

Reply:
I just used to what I assume to be the equivalent of `usize`. Apparently, using `size_t` for loop counting and array indexing is common.

Reply:
There's a deep theological discussion to be had here but long story short: `usize` in Rust doesn't neatly map onto C's integer types (IMHO `usize` in Rust conflates notions that shouldn't be conflated... but that's another story). That said, on reflection, I think `size_t` is OK in this case.